analyses/fusion_filtering/run_fusion_merged.sh

```diff
@@ -46,11 +47,11 @@ normal_expression_adrenal_gland="${references_path}/gtex_adrenal_gland_TPM_hg38.
 normal_expression_brain="${references_path}/gtex_brain_TPM_hg38.rds"
 
 # independent sample list
-independent_RNA_primary="${data_path}/independent-specimens.rnaseq.primary.tsv"
-independent_RNA_relapse="${data_path}/independent-specimens.rnaseq.relapse.tsv"
+independent_RNA_primary="${indep_path}/independent-specimens.rnaseq.primary-pre-release.tsv"
+independent_RNA_relapse="${indep_path}/independent-specimens.rnaseq.relapse-pre-release.tsv"
```

Comment:
I think we might want to put these into v11 and still use the data folder to avoid module paths. Can you update those on s3 please?

Reply:
sure - done.